test/statvar_ranking/missing_Earth_country_rankings.json

```diff
@@ -554,18 +554,6 @@
     ],
     "relatedChart": {}
   },
-  {
-    "title": "Number of people employed",
-    "statsVars": [
-      "Count_Person_Employed"
-    ],
-    "relatedChart": {
-      "scale": true,
-      "statsVars": [
-        "Count_Person_Employed"
-      ]
-    }
-  },
```

Comment:
Is removing this test case expected?

Reply:
I think this means we now have this SV... "missing_....json" is the file